src/qtiCommonRenderer/helpers/sizeAdapter.js

```diff
@@ -61,5 +63,14 @@ export default {
                 );
             }, 1);
         });
+    },
+
+    /**
+     * Reset height to jQueryElement(s) to auto
+     *
+     * @param {jQueryElement|widget} target
+     */
+    resetSize(target) {
+        target.$container.find(itemSelector).height('auto');
```

Comment:
**thought:** It would have been better to add a `reset()` method to the helper `util/adaptSize` in order to keep consistency in the API. Then it could have been called there.

**suggestion:**
Something like:

```javascript
        adaptSize.resetHeight(target.$container.find(itemSelector));
```